src/components/simpleDropdown/simpleDropdown.tsx

```diff
@@ -50,15 +57,18 @@ const SimpleDropdown: React.FC<SimpleDropdownProps> = ({
           variant={variant}
           aria-label={ariaLabel}
           isDisabled={!dropdownItems || dropdownItems.length === 0}
+          {...(menuToggleOuiaId && {
+            'data-ouia-component-id': menuToggleOuiaId
```

Comment:
Alright, that appears to work as I wanted. Fixed in 979fe3812afac4434e397acb3cc584e40da9debc .

I searched the web to learn how to put attribute only if variable has value, and I found that React basically does that, but there is somewhat complex underlying logic that sometimes does not work as you would think. So I thought that instead of learning React internals, I'm just going to be explicit.

For the record, this "create and destruct object if variable is set" pattern is already used twice in [typeaheadCheckboxes.tsx](https://github.com/quipucords/quipucords-ui/blob/main/src/components/typeAheadCheckboxes/typeaheadCheckboxes.tsx#L170). I guess I can remove it since I already touch that file in this PR. Or I can leave it as is.

Reply:
understood

typeaheadCheckboxes.tsx is copied almost exactly from the PF org examples. generally, we shouldn't be doing that since example code isn't necessarily production ready. It's one of the components that needs to be heavily refactored